spyder/plugins/explorer/widgets.py

```diff
@@ -865,7 +901,7 @@ def vcs_command(self, fnames, action):
             QMessageBox.critical(self, _("Error"),
                 _("""<b>Unable to find external program.</b><br><br>%s""")
                     % to_text_string(msg))
-        

```

Comment:
Please don't remove unrelated blank lines or trailing spaces (we'll do it globally at some point) since it makes your PR harder to review and leads to merge conflicts with other PRs. Thanks.